front/src/modules/conflict/utils.ts

```diff
@@ -1,43 +1,74 @@
-import type { Conflict } from 'common/api/osrdEditoastApi';
+import { type Conflict } from 'common/api/osrdEditoastApi';
 import type { TimetableItemWithDetails } from 'modules/trainschedule/components/Timetable/types';
 import computeOccurrenceName from 'modules/trainschedule/helpers/computeOccurrenceName';
 import type { TimetableItemId } from 'reducers/osrdconf/types';
-import { formatEditoastIdToTrainScheduleId, formatEditoastIdToPacedTrainId } from 'utils/trainId';
+import {
+  formatEditoastIdToTrainScheduleId,
+  formatEditoastIdToPacedTrainId,
+  isPacedTrainWithDetails,
+} from 'utils/trainId';
 
 import type { ConflictWithTrainNames } from './types';
 
 function getConflictTrainNames(
   conflict: Conflict,
-  trainNameMap: Map<TimetableItemId, string>
+  trainMap: Map<TimetableItemId, TimetableItemWithDetails>
 ): string[] {
   const trainScheduleNames = conflict.train_schedule_ids.map((id) =>
-    trainNameMap.get(formatEditoastIdToTrainScheduleId(id))
+    trainMap.get(formatEditoastIdToTrainScheduleId(id))
```

Comment:
```suggestion
    trainMap.get(formatEditoastIdToTrainScheduleId(id))?.name
```
Here it's the name that should be returned

Reply:
Done